envs/.env.prod

```diff
@@ -0,0 +1,15 @@
+# This is the standard lower environment for Content API.
+NEXT_PUBLIC_DRUPAL_BASE_URL=https://prod.cms.va.gov/
+NEXT_IMAGE_DOMAIN=https://prod.cms.va.gov/
+
+NEXT_PUBLIC_ASSETS_URL=/generated/
+
+# for Drupal preview
+DRUPAL_PREVIEW_SECRET=secret
+# store these securely in the build pipeline env
+#DRUPAL_CLIENT_ID=Retrieve this from AWS SSM /cms/consumers/next-build/client_id
+#DRUPAL_CLIENT_SECRET=Retrieve this from AWS SSM /cms/consumers/next-build/client_secret
+
+# Google Analytics
+# These print directly to the page so do not need to store in SSM.
+NEXT_PUBLIC_GOOGLE_TAG_MANAGER_ID=GTM-WFJWBD
```

Comment:
Are Tag manager Auth and Preview not needed for Prod? I'm guessing no, but want to make sure.

Reply:
correct, they aren't required and they come through as undefined. 

content-build prod analytics for ref: https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/assets/js/google-analytics/vagovprod.js